include/miopen/config.h.in

```diff
@@ -78,11 +78,62 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
-#cmakedefine MIOPEN_AMDGCN_ASSEMBLER "@MIOPEN_AMDGCN_ASSEMBLER@"
-#cmakedefine HIP_OC_COMPILER "@HIP_OC_COMPILER@"
-#cmakedefine MIOPEN_HIP_COMPILER "@MIOPEN_HIP_COMPILER@"
-#cmakedefine MIOPEN_OFFLOADBUNDLER_BIN "@MIOPEN_OFFLOADBUNDLER_BIN@"
+#ifdef MIOPEN_COMPILER_FLAGS_V2 /* Flag to switch to MIOPEN Compiler Flag Approach V2*/
+  /* Include rocm-core header for get ROCm Install Path Method */
+  #include <rocm-core/rocm_getpath.h>
+
+  /* Include header for PATH_MAX */
+  #if defined(__linux__)
+  #include <linux/limits.h>
+  #endif /* __linux__ */
```

Comment:
This PR uses mixed approach for indentation. Either use indentation within `#if...#endif` or no indentation -- but everywhere.

Reply:
Thanks for the comments, existing case updated in latest version, will make a note for future reference